ch03.asciidoc

```diff
@@ -289,17 +289,22 @@ While this method could be fully executed faster than the good and the bad way t
 === Invoices
 
 Every payment on the Lightning Network starts with a person who wants to receive bitcoins issuing an invoice.
+An invoice is nothing more than a very simple payment instruction containing information such as a unique payment identifier, called a payment hash, a recipient and an amount.
```

Comment:
Also contains other extra details which may be needed in order to complete the payment. One example if the `payment_addr` field for MPP payments.

Reply:
adjusted text to make it more general and open-ended.